models/agent_work_time/int_zendesk__ticket_work_time_business.sql

```diff
@@ -50,7 +50,7 @@ with ticket_historical_status as (
       status_valid_starting_at,
       status_valid_ending_at,
     ({{ dbt.datediff(
-            "cast(" ~ dbt_date.week_start('ticket_status_crossed_with_schedule.status_schedule_start','UTC') ~ "as " ~ dbt.type_timestamp() ~ ")", 
```

Comment:
the `'UTC'` setting doesn't appear to actually do anything for our use per [these lines](https://github.com/calogica/dbt-date/blob/main/macros/calendar_date/week_start.sql#L2-L3) from the original macro.